specs/protocol/holocene/predeploys.md

```diff
@@ -255,6 +258,21 @@ The following functions are updated to read from the `L1Block` contract by calli
 - `otherBridge()(address)`
 - `OTHER_BRIDGE()(address)`
 
+### GasPriceOracle
+
+In order to maintain accurate offchain fee estimation, the `GasPriceOracle` must be updated to allow users
+to estimate the operator fee.
+
+#### Interface
+
+##### `getOperatorFee`
+
+This function calculates the operator fee based on the expected amount of gas used for a certain transaction.
+
+```function
+function getOperatorFee(uint256 gasUsed)(uint256)
+```
+
```

Comment:
We will also need an `isHolocene` bool as well as branching logic in the fee estimation functions